components/api-gateway-migrator/README.md

```diff
@@ -0,0 +1,93 @@
+# API Gateway Migrator
+
+This tool translates the existing API objects (APIs) to new ones (APIRules).
+
+## Usage:
+To migrate APIs to APIRules, run:
+```bash
+go run cmd/main.go --label-blacklist=migration/status
+```
+
+To see all possible arguments, run:
+```bash
+go run cmd/main.go -h
+```
+
+After migration, the host is randomized to point to a non-existing location. This disables the existing APIs, but does not remove the objects. To delete all migrated API objects, run:
+```bash
+kubectl delete apis -l migration/status=migrated --all-namespaces
+```
+
+## Development
+### API migration example
+
+
+This example shows a sample input API:
+    ```yaml
```

Comment:
There is a problem with the rendering of these two code snippets - I think the indentation can be the issue

Reply:
Done.